lib/cn/kvset.c

```diff
@@ -373,41 +370,9 @@ kvset_kblk_init(
 }
 
 static merr_t
-vblock_udata_init(
-    struct mbset *       mbs,
-    uint                 bnum,
-    uint *               argcp,
-    u64 *                argv,
-    struct mblock_props *props,
-    void *               rock)
-{
-    return vbr_desc_read(
-        mbset_get_mp(mbs),
-        mbset_get_map(mbs),
-        bnum,
-        argcp,
-        argv,
-        props,
-        rock);
-}
-
-static merr_t
-vblock_udata_update(
-    struct mbset *       mbs,
-    uint                 bnum,
-    uint *               argcp,
-    u64 *                argv,
-    struct mblock_props *props,
-    void *               rock)
-{
-    return vbr_desc_update(
-        mbset_get_mp(mbs),
-        mbset_get_map(mbs),
-        bnum,
-        argcp,
-        argv,
-        props,
-        rock);
+vblock_udata_init(const struct kvs_mblk_desc *mblk, void *rock)
```

Comment:
Delete this function

Reply:
It wraps the call to `vbr_desc_read` which has a slightly different signature.  If I use `vbr_desc_read` as a param to `mbset_create`, the compiler complains of the type mismatch.

Reply:
Oh, I didn't see that. I would probably prefer a cast, but I will let you make the call!

Reply:
It would be a function signature cast, which would hide the fact that the underlying cast is simple: from `void*` to `struct vblock_desc *`.  If I used a function signature cast, I could pass it `realloc` and nobody would know.